.eslintrc.json

```diff
@@ -1,27 +1,26 @@
 {
-    "env": {
-      "browser": true,
-      "es6": true,
-      "jest": true
+  "env": {
+    "browser": true,
+    "es6": true,
+    "jest": true
+  },
+  "parser": "@babel/eslint-parser",
+  "parserOptions": {
+    "ecmaFeatures": {
+      "jsx": true
     },
-    "parser": "@babel/eslint-parser",
-    "parserOptions": {
-      "ecmaFeatures": {
-        "jsx": true
-      },
-      "ecmaVersion": 2018,
-      "sourceType": "module"
-    },
-    "extends": ["airbnb", "plugin:react/recommended"],
-    "plugins": ["react"],
-    "rules": {
-      "react/jsx-filename-extension": ["warn", { "extensions": [".js", ".jsx"] }],
-      "react/react-in-jsx-scope": "off",
-      "import/no-unresolved": "off",
-      "no-shadow": "off"
-    },
-    "ignorePatterns": [
-      "dist/",
-      "build/"
-    ]
-  }
\ No newline at end of file
+    "ecmaVersion": 2018,
+    "sourceType": "module"
+  },
+  "rules": {
+    // "react/jsx-filename-extension": ["warn", { "extensions": [".js", ".jsx"] }],
+    // "react/jsx-filename-extension": [1, { "extensions": [".js", ".jsx"] }],
+    "react/react-in-jsx-scope": "off",
```

Comment:
Hello, @butlermuwo, it is not good practice editing the linter files to your liking. I suggest that we mantain the rules set and try to improve the code base.

- [ ] Please kindly make the required changelles

Reply:
Hi Cathella! I have maintained the codebase as you suggested, when I run eslint locally the linters test is passing except on Github kindly help me solve this, I have tried reaching out for help and I have done some research online but I can't get conclusive responses to slove this problem.